### PR TITLE
[Cypress] Create method for checking project import completion

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create method for checking project import completion

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/12991

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
ci-build

File: cypress-tests/cypress/pageobjects/ide/ProjectTree.ts [hunk 1]
```diff
@@ -136,5 +159,64 @@ export class ProjectTree {
         this.clickOnItem(`${pathToItem}/${fileName}`)
     }
 
+    waitProjectImported(projectName: string, rootSubitem: string) {
+        cy.log("**=> ProjectTree.waitProjectImported**")
+            .then(() => {
+                let attempts: number = 4;
+                let pollingEvery: number = 30000;
```
Comment (dmytro-ndp) [inline cypress-tests/cypress/pageobjects/ide/ProjectTree.ts]: What about having such kind of numbers defined as constants in one place?

File: cypress-tests/cypress/pageobjects/ide/ProjectTree.ts [hunk 2]
```diff
@@ -136,5 +159,64 @@ export class ProjectTree {
...
+                let currentAttempt: number = 1;
+
+                this.waitImported(projectName, rootSubitem, attempts, currentAttempt, pollingEvery)
+            })
+    }
+
+    private waitImported(projectName: string, rootSubitem: string, attempts: number, currentAttempt: number, pollingEvery: number): Promise<void> {
```
Comment (dmytro-ndp) [inline cypress-tests/cypress/pageobjects/ide/ProjectTree.ts]: waitImported could be confusing.
`doWaitProjectImported` might be clear here.

File: cypress-tests/cypress/pageobjects/ide/ProjectTree.ts [hunk 3]
```diff
@@ -136,5 +159,64 @@ export class ProjectTree {
...
+        return new Promise((resolve, reject) => {
+            let rootItem: string = `/${projectName}`;
+
+            this.expandItem(rootItem)
+            this.waitItemExpanded(rootItem)
+
+            cy.get('body')
+                .then(body => {
+                    let rootItemLocator: string = this.getTreeItemLocator(`/${projectName}`);
```
Comment (dmytro-ndp) [inline cypress-tests/cypress/pageobjects/ide/ProjectTree.ts:182]: What about extracting enclosed code into the external method to simplify readability and make it possible to reuse it?

File: cypress-tests/cypress/utils/workspace/TestWorkspaceUtil.ts [hunk 4]
```diff
@@ -16,8 +16,8 @@ import { Promise, resolve, reject } from "bluebird";
 export class TestWorkspaceUtil {
 
     private waitRunningStatus(workspaceNamespace: string, workspaceName: string, attempt: number): Promise<void> {
-        const maximumAttempts: number = 300;
-        const delayBetweenAttempts: number = 5000;
+        const maximumAttempts: number = 15;
```
Comment (dmytro-ndp) [inline cypress-tests/cypress/utils/workspace/TestWorkspaceUtil.ts]: The same comment as https://github.com/eclipse/che/pull/13023/files#r270416462

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Review by musienko-maxim — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Review by dmytro-ndp — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by Ohrimenko1988** [comment]
ci-build

**Pull request merged**